bids/variables/kollekshuns.py

```diff
@@ -53,8 +54,18 @@ def __init__(self, variables):
                              "variables at more than one level of analysis. "
                              "Levels found in input variables: %s" %
                              var_levels)
-
-        self.level = list(var_levels)[0]
+        elif not var_levels:
+            raise ValueError(
+                "None of the provided variables matched any of the known levels, which are: %s"
+                % (', '.join(sorted(SOURCE_TO_LEVEL.values())))
+            )
+
+        try:
+            self.level = list(var_levels)[0]
+        except:
+            import pdb;
+            pdb.set_trace()
+            raise
```

Comment:
Oopsy - this portion of the change should be removed ;-)

Reply:
sorry - I think I removed it now (?)